web/docusaurus.config.js

```diff
@@ -280,6 +282,18 @@ module.exports = {
             from: '/docs/tutorials/todo-app',
             to: '/docs/tutorial/create',
           },
+          {
```

Comment:
Nice job on grouping migrations in top level category and collapsing it by default!